db/migrate/20201113001940_add_project_id_to_campaigns.rb

```diff
@@ -0,0 +1,7 @@
+class AddProjectIdToCampaigns < ActiveRecord::Migration[6.0]
+  def change
+    add_column :campaigns, :project_id, :bigint, null: true
+    add_index :campaigns, :project_id
+    add_foreign_key :campaigns, :projects
```

Comment:
fyi `add_reference :campaigns, :project, null: true, foreign_key: true` does the same thing all in one line